rasac_osw:
  ros_parameters:
    min_rad: 0
    Max_rad: 3.14159263
    Max_loop: 100
    threshould: 0.5
    min_line_particles: 100 # それっぽい直線であると認識する最低の点の数
